frontend/src/screens/learner/details.tsx

```diff
@@ -193,7 +196,9 @@ export const StudyDetails: React.FC<{ studies: ParticipantStudy[] }> = ({ studie
         }
     }, [study])
 
-    if (!study) return null
+    if (isLoading) return null
+
+    if (!study) return <Navigate to="/studies" />
```

Comment:
Changes in this file are due to navigating directly into the details page, the studies weren't loaded yet and it wasn't rendering the component in time, i like the idea of fetching the individual study too on opening in case something with the study changes, but i know there is also an argument for filtering it out through the main studies list 🤔